### PR TITLE
Development branch 1 into main 

### PR DESCRIPTION
**Pull request by Chris-cvR**
@mircealungu We have made some more changes and fixes as requested based on the comments from the last review :)

### CODE REVIEW

**Comment by Chris-cvR** [comment]
@mircealungu I accidentally made a mistake and merged a branch into the main instead of development. We made the changes you requested and will now keep working from a new development branch

**Comment by mircealungu** [comment]
@Chris-cvR - you don't have to work from a different development branch. You should be able to work from the same one. Just rebase it from master. Do you know what that means? If not, ask a question on LearnIt ;)

**Comment by mircealungu** [comment]
And good that you made the changes! 

And for the future, to make sure that you can't make the merge to master by mistake, protect it from merge w/o a code review. That should prevent this situation in the future.

**Comment by Chris-cvR** [comment]
@mircealungu We have since then committed things to the development branch and would a git rebase not overwrite these changes? I am also not sure about how to do a git rebase, I havent tried it before. I will ask on LearnIT :)

File: src/assets/images/climb_cliff.png.jpg.webp
Comment (mircealungu) [inline src/assets/images/climb_cliff.png.jpg.webp:1]: name of this file is a bit funny. 
the extension should tell you what kind of file you have.
the formats are exclusive. a file can't be both png and jpg and webp

File: src/components/common/Comment.js [hunk 1]
```diff
@@ -9,19 +10,29 @@ function Comment() {
   // Function to add a new comment to the list
   const addComment = () => {
     if (newComment) {
-      setComments([...comments, newComment]); // Update the list of comments with the new comment
+      // Create comment with assigning id and text to each. Maybe we need to have title here as well?
+      const commentObject = {
+        id: Date.now(), // Timestamp as id - Not sure if we want this, but I figured it might make sense in this case
```
Comment (mircealungu) [inline src/components/common/Comment.js:15]: maybe use [nanoid](https://www.npmjs.com/package/nanoid)?

**Review by mircealungu — CHANGES_REQUESTED** [review]
a few changes would be good.

**Pull request closed without merging**